src/components/Categories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      <h1>Under construction</h1>
```

Comment:
- Please change this to a button with the text `Check status`. This sets up the project nicely for upcoming changes.